docs/hubs-cloud-faq.md

```diff
@@ -0,0 +1,29 @@
+---
+id: hubs-cloud-faq
+title: Hubs Cloud FAQ
+sidebar_label: FAQ
+---
+
+## Personal vs. Enterprise?
+
+- Personal defaults are lower cost. Enterprise defaults are higher cost but improve site functionality like no database pausing.
+- The main difference: Enterprise allows multiple servers. If one goes down unexpectantly, the others will take over. Personal allows only one server.
+- Beyond that, they are the same in the options they offer and cost per server is the same. Personal can be configured to be an expensive, highly scalable single server, and vice versa.
+
+## How to change my server size up from a t3.micro to c4.large?
```

Comment:
Also "Can I change the # of servers I am running?" (yes, for Enterprise do update stack, no for Personal)